system/ee/ExpressionEngine/Service/ChannelSet/Set.php

```diff
@@ -426,9 +438,13 @@ private function getFieldByName($field_name)
      */
     private function loadUploadDestinations($destinations)
     {
+        // $site_name = ee()->config->item('site_name');
+
         foreach ($destinations as $upload_data) {
             $destination = ee('Model')->make('UploadDestination');
             $destination->site_id = $this->site_id;
+            // $destination->server_path = sprintf($destination->server_path, $site_name);
+            // $destination->url = sprintf($destination->url, $site_name);
```

Comment:
If the commented lines are not needed, can those just be remomoved?

Reply:
My bad, @intoeetive !